src/components/__tests__/NavigationData.spec.js

```diff
@@ -19,4 +20,20 @@ describe('NavigationData.vue', () => {
   it('renders BaseList component', () => {
     expect(wrapper.findAll(BaseList)).toHaveLength(1)
   })
+
+  it('renders proper number of items', () => {
+    expect(wrapper.findAll('[data-test-base-list-item]')).toHaveLength(3)
+  })
+
+  it('contains Course item', () => {
+    expect(wrapper.text()).toContain('90°')
```

Comment:
What do you think of asserting `${report.course}°` here? That way, the values are not bound to this line, but to the real thing we're testing.